contributing/index.md

```diff
@@ -94,16 +131,6 @@ legacssy: {
 {% endhighlight %}
 
 
-## Contributing to existing components
-
-If you'd like to contribute to an existing Capital Framework component, please use the [fork and pull model](https://help.github.com/articles/using-pull-requests/).
-
-## Creating new Components
-
-We hope to soon offer a Yeoman generator for creating new components quickly. In the meantime our current recommendation is to clone and edit an existing component.
```

Comment:
We could still do this, in theory, even in the single-repo model. @contolini I'll be curious to hear your feedback on the new component process after working with Chuck to add cf-tables.